### PR TITLE
Block/Allow service on destination

### PR DESCRIPTION
**Pull request by bodnara**
* added buttons for blocking/allowing service on destination to facility detail - service destination page

### CODE REVIEW

File: apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html [hunk 1]
```diff
@@ -15,6 +15,24 @@ <h1 class="page-subtitle">{{'FACILITY_DETAIL.SERVICES_DESTINATIONS.TITLE' | tran
     [disabled]="selected.selected.length === 0"  (click)="removeDestination()">
     {{'FACILITY_DETAIL.SERVICES_DESTINATIONS.REMOVE' | translate}}
   </button>
+  <button
+    *ngIf="!loading && allowAuth"
+    mat-flat-button
+    color="accent"
```
Comment (Vojtech-Sassmann) [inline apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html:21]: I don't really like the style of these buttons. You should usually have only one action and one danger button at most. This really makes it look weird. I would just use a neutral color for these two buttons.

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]

**Review by xkureck — COMMENTED** [review]
I agree with Vojta
and second thing if all things that are selected are allowed, the button "Allow" make disabled, if all thing that are selected are blocked, the button "Block" should be disabled, if the things that are selected are mixed make the both buttons non disabled
image
my idea how the buttons could look

File: apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html [hunk 2]
```diff
@@ -15,6 +15,28 @@ <h1 class="page-subtitle">{{'FACILITY_DETAIL.SERVICES_DESTINATIONS.TITLE' | tran
     [disabled]="selected.selected.length === 0"  (click)="removeDestination()">
     {{'FACILITY_DETAIL.SERVICES_DESTINATIONS.REMOVE' | translate}}
   </button>
+  <button
+    *ngIf="!loading && allowAuth"
+    mat-flat-button
```
Comment (xkureck) [inline apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html:20]:
```suggestion
    mat-stroked-button
```

File: apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html [hunk 3]
```diff
@@ -15,6 +15,28 @@ <h1 class="page-subtitle">{{'FACILITY_DETAIL.SERVICES_DESTINATIONS.TITLE' | tran
...
+    [disabled]="selected.selected.length === 0 || allSelectedAllowed()"
+    (click)="onAllow()">
+    <mat-icon>
+      check_circle_outline
+    </mat-icon>
+    {{'FACILITY_DETAIL.SERVICES_DESTINATIONS.ALLOW' | translate}}
+  </button>
+  <button
+    *ngIf="!loading && blockAuth"
+    mat-flat-button
```
Comment (xkureck) [inline apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-services-destinations/facility-services-destinations.component.html:31]:
```suggestion
    mat-stroked-button
```

**Review by xkureck — COMMENTED** [review]
In my opinion the "flat" buttons looks really weird when they doesnt have any colour

**Review by xkureck — APPROVED** [review]

**Pull request merged**